edifact-edi/2.1/modules/ROOT/pages/index.adoc

```diff
@@ -448,14 +450,14 @@ Individual messages have their own maps, with the following keys:
 [%header%autowidth.spread]
 |===
 |Key |Description
-|Detail |Map of segments or loops from the detail section of the message. Values are maps for segments or loops which occur at most once, lists of maps for values which may be repeated.
+|Detail |Map of segments or loops from the detail section of the message. Values are maps for segments or loops which occur at most once, lists of maps for values which can be repeated.
```

Comment:
I don't really understand the text here. Splitting the sentence into two sentences might help.

which > that